src/components/ArticleComments.jsx

```diff
@@ -8,14 +8,15 @@ export const ArticleComments = ({article}) => {
 
     const [comments, setComments] = useState([])
     const [loading, setLoading] = useState(true);
+    const [hasDeleted, setHasDeleted] = useState(false)
 
     useEffect(() => {
         getCommentsByArticleId(article.article_id)
         .then((result) => {
             setComments(result)
             setLoading(false)
         })
-    }, [])
+    }, [hasDeleted])
```

Comment:
It seems you have created this has deleted state but never change it. I would potentially remove it as we don't want to re-fetch our comments each time we delete :)